airbyte-scheduler/scheduler-persistence/src/main/java/io/airbyte/scheduler/persistence/job_error_reporter/JobErrorReporter.java

```diff
@@ -118,9 +118,7 @@ public void reportSourceCheckJobFailure(final UUID sourceDefinitionId,
     final StandardSourceDefinition sourceDefinition = configRepository.getStandardSourceDefinition(sourceDefinitionId);
     final Map<String, String> metadata = MoreMaps.merge(
         getSourceMetadata(sourceDefinition),
-        Map.of(
-            JOB_ID_KEY, jobContext.jobId().toString(),
-            CONNECTOR_COMMAND_META_KEY, "check"));
```

Comment:
Connector command will now be already set on the FailureReason as metadata, so don't need to set it here when reporting the error